src/EditorFeatures/Test/Rename/RenameRewriterTests.Verifier.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Rename;
+using Microsoft.CodeAnalysis.Rename.ConflictEngine;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+using Xunit;
+using RenameAnnotation = Microsoft.CodeAnalysis.Rename.ConflictEngine.RenameAnnotation;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
+{
+    public partial class RenameRewriterTests
+    {
+        private const string ConflictTag = "Conflict";
+        private const string RenameTag = "Rename";
+
+        protected sealed class Verifier : IDisposable
```

Comment:
I am mostly testing the rename rewriter here.
So the existing renamer test is testing with one renaming symbol passed into Renamer ->  RenameLocations --> ConflictResolution ---> Renamerewriter.
And there are many places assuming there is only one renaming symbol. 
I started with changing the rename rewriter so that's why we have this test verifier.

But yes, in the end, the existing rename tests would be modified so that it would test these